tests/gem5/asmtest/README.md

```diff
@@ -0,0 +1,9 @@
+# ASM Test
+
+These tests run a set of RISCV binaries on a bare bones syscall execution.
+In addition, these test run these binaries against different CPU types.
+To run these tests by themselves, you can run the following command in the tests directory:
+
+```bash
+./main.py run gem5/asmtest --length=[length]
```

Comment:
What does `--length` mean in this case?

I guess this is copy pasted across all of these readme's, but does it make sense for this one?

I suggest listing the tests in these readme and saying whether they are quick/long/very long.   HOWEVER, I suggest doing that in another change 😄